VTUL/dlp-access
# just use searchbar instead of homepageSearchBar

**Refactor to just use SearchBar.js instead of HomepageSearchBar.js**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2139) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Refactor to just use SearchBar.js instead of HomepageSearchBar.js

# What's the changes? (:star:)
* Refactor to just use SearchBar.js instead of HomepageSearchBar.js

# How should this be tested?
* Check if search from the homepage still works.
* Check if site breadcrumbs update correctly on search results page when searching from homepage

# Additional Notes:
* branch: `LIBTD-2139`

# Interested parties
@yinlinchen 

(:star:) Required fields
